content/azs.md

```diff
@@ -146,150 +202,223 @@ Deleting unneeded instances dummy: dummy/7f30aae8-9f03-4b0b-88a1-2d0ab8a78fba (1
+1. Create a new instance (VM) in `z2` so that “_new instances will be spread
+   as evenly as possible over specified AZs_”.
+2. Delete a currently serving instance (VM) in `z1` so that “_instances will
+   be rebalanced ... to even out distribution_”
 
 
-### Scenario: the instance_group uses persistent disks:
+#### Scenario #2: the instance group uses persistent disks
+
+Given such a `dummy` instance group defined in a Bosh deployment manifest:
 
 ```
 instance_groups:
-- name: dummy
-  azs:
-  - az1
-  - az3
-  instances: 3
-  persistent_disk: 1024
-…
+  - name: dummy
+    azs:
+      - z1
+      - z3
+    instances: 3
+    persistent_disk: 1024
+# ...
 ```
-##### bosh vms
+
+And Bosh instances (VMs) spread over the above availability zones like this:
+
 ```
-Instance                                   ... AZ  
-dummy/3697cb63-5329-4b61-8251-6acd73fe5d8b ... az3 
-dummy/63d450fc-a071-4e19-b0ba-c8fdb147dcce ... az1 
-dummy/528993ea-5e8b-4d7f-8844-98234bcb0575 ... az1 
+$ bosh instances
+Instance                                   ... AZ
+dummy/3697cb63-5329-4b61-8251-6acd73fe5d8b ... z3
+dummy/63d450fc-a071-4e19-b0ba-c8fdb147dcce ... z1
+dummy/528993ea-5e8b-4d7f-8844-98234bcb0575 ... z1
 ```
-##### bosh deploy ...
+
+Then after adding the new `z2` availability zone to the instance group, the
+`bosh deploy` operation will behave as follows:
+
 ```
 Using deployment 'dummy'
 
   instance_groups:
   - name: dummy
     azs:
-+   - az2
++   - z2
 ...
 Updating instance dummy: dummy/63d450fc-a071-4e19-b0ba-c8fdb147dcce (0) (canary)
 Updating instance dummy: dummy/528993ea-5e8b-4d7f-8844-98234bcb0575 (1)
 Updating instance dummy: dummy/3697cb63-5329-4b61-8251-6acd73fe5d8b (2)
 ```
 
-##### Outcome:
+And after this is done, the resulting instances (VMs) will be placed like
+this:
+
 ```
-Instance                                   ... AZ  
-dummy/3697cb63-5329-4b61-8251-6acd73fe5d8b ... az3 
-dummy/63d450fc-a071-4e19-b0ba-c8fdb147dcce ... az1 
-dummy/528993ea-5e8b-4d7f-8844-98234bcb0575 ... az1 
+$ bosh instances
+Instance                                   ... AZ
+dummy/3697cb63-5329-4b61-8251-6acd73fe5d8b ... z3
+dummy/63d450fc-a071-4e19-b0ba-c8fdb147dcce ... z1
+dummy/528993ea-5e8b-4d7f-8844-98234bcb0575 ... z1
 ```
 
-Since the the above scenario does utilize persistent disks, adding az2 to the list of azs will:
+Since the the above scenario makes use of persistent disks, adding `z2` to the
+list of availability zones will skip redeploying an instance (VM) from `z1` to
+`z2`, in order to avoid recreating the persistent disk that could produce
+prejudiciable data loss, depending on the deployed technology.
 
-1. Skip redeploying a vm from az1 to az2 to avoid recreating the persistent disk and potentially incur dataloss.
+!!! Note
+    Bosh is not aware about the capabilities, in terms of distributed state,
+    of the software it deploys though. For some distributed software, loosing
```

Comment:
typo `losing`

Reply:
+1 for having a "Migrating from / Differences with v1" and not placing it so prominently.